src/ws/cockpit.socket.in

```diff
@@ -1,11 +1,12 @@
 [Unit]
 Description=Cockpit Web Service Socket
 Documentation=man:cockpit-ws(8)
+Wants=cockpit-motd.service
 
 [Socket]
 ListenStream=9090
-ExecStartPost=-/bin/ln -sf @datadir@/@PACKAGE@/issue/active.issue /run/cockpit/issue
-ExecStopPost=-/bin/ln -sf @datadir@/@PACKAGE@/issue/inactive.issue /run/cockpit/issue
+ExecStartPost=-@datadir@/@PACKAGE@/motd/activate-motd
+ExecStopPost=-@datadir@/@PACKAGE@/motd/deactivate-motd
```

Comment:
Matter of preference/nitpicking of course, but I feel like it's a lot of overhead to ship two new scripts, do the macro substition, and do this indirection through a shell just for one and two commands. You can just directly put the commands here, and use two `ExecStartPost=` commands. This unit already does all the `@` substition.

Reply:
In retrospect (and particularly since it's possible to put two commands) I agree.